wgsl/index.bs

```diff
@@ -988,6 +988,8 @@ file.
 The input and output parameters to the entry point are determined by which
 global variables are used in the function and any called functions.
 
+The entry point function must be declared with no arguments and returning `void`.
```

Comment:
much related to  https://github.com/gpuweb/gpuweb/issues/889#issuecomment-654370178 :)